### PR TITLE
[FEAT]: implement unregister method

### PR DESCRIPTION
**Pull request by gloz0315**
### What is the PR type?
- [x] : feature
- [ ] : refactor
- [ ] : bug
- [ ] : documentation
- [ ] : chore
- [ ] : displace
- [ ] : etc

### What does this PR do?

- implement unregister method

### CODE REVIEW

**Review by devJWL — APPROVED** [review]
LGTM

File: src/main/java/com/ptjcoding/nbcampspringnewsfeed/domain/blacklist/service/BlackListService.java [hunk 1]
```diff
@@ -5,4 +5,6 @@
 public interface BlackListService {
 
   void register(BlackListRequestDto dto);
+
+  void unregister(BlackListRequestDto dto);
```
Comment (ruh0n) [inline src/main/java/com/ptjcoding/nbcampspringnewsfeed/domain/blacklist/service/BlackListService.java]: deregister이 더 적절한 표현 같습니다.

https://grammarhow.com/unregister-vs-deregister/
https://english.stackexchange.com/questions/25931/unregister-vs-deregister

**Review by ruh0n — APPROVED** [review]
LGTM

File: src/main/java/com/ptjcoding/nbcampspringnewsfeed/domain/blacklist/service/BlackListService.java (hunk 1 above)
Comment (gloz0315) [reply src/main/java/com/ptjcoding/nbcampspringnewsfeed/domain/blacklist/service/BlackListService.java]: 수정했습니다~

**Pull request merged**